package/MDAnalysis/auxiliary/base.py

```diff
@@ -283,13 +298,15 @@ class AuxReader(metaclass=_AuxReaderMeta):
                       'time_selector', 'data_selector', 'constant_dt', 'auxname',
                       'format', '_auxdata']
 
-    def __init__(self, represent_ts_as='closest', auxname=None, cutoff=-1,
+    def __init__(self, represent_ts_as='closest', auxname=None, cutoff=None,
                  **kwargs):
         # allow auxname to be optional for when using reader separate from
         # trajectory.
         self.auxname = auxname
         self.represent_ts_as = represent_ts_as
-        self.cutoff = cutoff
+        # no cutoff was previously passed as -1. This check is to maintain this
+        # behaviour: negative number is appropriately turned to None
```

Comment:
Note that this is not quite the old behavior: we now use any negative number as `None`. Previously only -1 would be seen as default. Arguably, that was a bug and we would have gotten weird behavior for -2. 

Therefore, I support the change as is.

I think you can add a CHANGELOG entry for fixes that you removed undefined behavior for negative numbers <-1 for cutoff.